code/__HELPERS/dynamic_human_icon_gen.dm

```diff
@@ -14,6 +14,7 @@ GLOBAL_LIST_EMPTY(dynamic_human_appearances)
 	dummy.underwear = "Nude"
 	dummy.undershirt = "Nude"
 	dummy.socks = "Nude"
+	dummy.bra = "Nude"
```

Comment:
Any code that you add in non-modular files (so they aren't under the `modular_skyrat` directory) needs to have an associated modularization comment in the form of `// SKYRAT EDIT (ADDITION if it's something you add, REMOVAL if it's something that's being removed, nothing for an edit) - Brief explanation of your change (a few words, usually the name of your PR, in this case, Underwear and bra split (- ORIGINAL: original line of code here, if it's just an edit and not an addition/removal)`

Reply:
So in this case, it'd be like this:
```suggestion
	dummy.bra = "Nude" // SKYRAT EDIT ADDITION - Underwear and bra split
```